### ISSUE TITLE
PR #42 review

### ISSUE THREAD

**Issue by ashes999** [openfl/svg#44]
[This PR](https://github.com/openfl/svg/pull/42) was merged in, but wasn't reviewed. It still needs a review. @ibilon

**Comment by ibilon** [comment]
Yeah sorry, been busy. Right now I'm at the WWX, I'll look at the PRs when I get back.
